packages/flutter_driver/lib/src/gesture.dart

```diff
@@ -5,12 +5,15 @@
 import 'message.dart';
 import 'find.dart';
 
+/// Taps on a target widget located by [finder].
 class Tap extends CommandWithTarget {
   @override
   final String kind = 'tap';
 
+  /// Creates a tap command to tap on a widget located by [finder].
   Tap(SerializableFinder finder) : super(finder);
 
+  /// Deserializes this command from JSON.
```

Comment:
Anywhere where you currently say "Deserializes this command from JSON.", you should either describe the expected shape of the JSON input, or point to the API that generates the expected JSON in the first place.